app.py

```diff
@@ -110,7 +127,9 @@ def start_item(item_id):
     @app.route('/complete/<item_id>', methods=['POST'])
     @login_required
     def complete_item(item_id):
+        app.logger.debug("User performing complete")
         user_role = get_user_role()
+        app.logger.info("Value of user_role is %s", user_role)
```

Comment:
Rather than printing the user role here as info, I'd be more inclined to switch this to a warning & print this out only if they don't pass the role check - this log doesn't tell us a lot if the user manages to complete an item but is important if we discover that a particular role keeps trying to perform an action and failing (we might stop them trying to do it, or realise that we've incorrectly denied them)

Reply:
done